### PR TITLE
rm: boards: add imx8mp + se050 page

### PR DESCRIPTION
**Pull request by vanmaegima**
Signed-off-by: Vanessa Maegima <vanessa.maegima@foundries.io>

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 3b0a9c696c347ec78b5af0e1bd629b430b5997c0 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1398/docs/artifacts/html/index.html

File: source/reference-manual/boards/se050_imx8mp.rst [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+Be sure that the jumpers on the SE050 evaluation board are
+set as follows:
+
+.. figure:: /_static/boards/se050ard_jumpers.png
+     :width: 400
+     :align: center
+
+     SE050 Jumper Settings
+
+Lastly the connected boards should look like this:
```
Comment (kprosise) [inline source/reference-manual/boards/se050_imx8mp.rst:60]:
```suggestion
Lastly, the connected boards should look like this:
```

**Review by kprosise — APPROVED** [review]
One suggestion, but it LGTM!

**Review by ricardosalveti — APPROVED** [review]
With Katrina's suggestions, LGTM

**Comment by doanac** [comment]
Docs for 6c3763352d4d037937d521af7535ea86a16b393a are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1399/docs/artifacts/html/index.html

**Review by MrCry0 — APPROVED** [review]

**Review by angolini — APPROVED** [review]
LGTM

**Pull request merged**